### DISCUSSION TITLE
Kotest test names shown as `invokeSuspend` for `FunSpec`

### DISCUSSION THREAD

**Discussion by Dogacel** [LemonAppDev/konsist#660] [Report a Problem] [answered]
### Tell us what version of Konsist you use.

0.12.2

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

When I write the konsist tests in the `FunSpec` form such as,

```kotlin
class CodeStyle : FunSpec({
  test("MVC architecture") {
   Konsist.scopeFromProject().checkArchitecture() // Do the rest
  }
})
```

The exception shown says the rule `invokeSuspend` is failing. When I use something else like `AnnotationSpec`, it works fine. But I personally want to enforce a single type of `Spec` generation, `FunSpec` so it is contradicting with my own code 😄 

### Actual behaviour

Supression for test names don't work because test name is `invokeSuspend`.

### Expected behaviour

Test name should be the determined correctly.

**Comment by igorwojda** [comment, marked as answer]
We have a fix for this. Will be released soon with `v0.13.0`.

**Reply by Dogacel** [reply]
Thanks, that's great 😄

**Comment by igorwojda** [comment]
A new version v0.13.0 has been released. It contains the fix.
See 
- https://github.com/LemonAppDev/konsist/releases/tag/v0.13.0
- https://docs.konsist.lemonappdev.com/getting-started/getting-started/kotest-support
